services/QuillLMS/app/services/google_integration/google_id_transferrer.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module GoogleIntegration
+  class GoogleIdTransferrer < ApplicationService
+    attr_reader :calling_user_id, :from_user, :from_user_google_id, :to_user, :to_user_google_id
+
+    def initialize(calling_user_id, from_user, to_user)
+      @calling_user_id = calling_user_id
+      @from_user = from_user
+      @from_user_google_id = from_user.google_id
+      @to_user = to_user
+      @to_user_google_id = to_user.google_id
+    end
+
+    def run
+      return if from_user_google_id.nil?
+
+      unlink_from_user
```

Comment:
Should we consider a `transaction` here? For example, would it be bad if the `unlink` worked and the `link` didn't?

Reply:
Good point.  This definitely has the feel of a bundled pair of actions.